src/main/java/softdesign/image/ImageRepository.java

```diff
@@ -1,27 +0,0 @@
-package main.java.softdesign.image;
```

Comment:
Can this be put back into the `image` package?

Reply:
I would prefer not at least not till the end because then i have to figure out how to run it with different directories within the command line

Reply:
Well at least put them back before right before merging to master so our code is consistent with the class diagram.

As for deliverable 3, I'm really against adjusting the code base for an editor. Tools should not dictate the code base but rather help you in working with the code base. So you should either:
1. Learn how to work with package in the command line
2. Invest in an IDE (I'd recommend IntelliJ)
3. Move the classes out of their package while working in a branch and put them back in their packages once you're done right before merging.

Reply:
Its definitely not gonna be number 2 and yeah I wanted to figure out how to do it, just didnt think it was number 1 priority right now